### DISCUSSION TITLE
Filelog receiver,  key_value_pair parser, how to use "tab" as pair delimiter

### DISCUSSION THREAD

**Discussion by owensulei3841** [open-telemetry/opentelemetry-collector-contrib#30558] [Q&A] [answered]
Hi everyone,
I have spring app that generate access logs, the access log has a bunch of key value pair like a=b c=d
My question is since my access log use "tab" instead of "whitespace" as pair delimiter, how can I change the default whiltespace pair delimiter to "tab" ? Thank you

Dear @BinaryFissionGames , I see you were working on filelog receiver, can I ask do you know how to do that? Thank you!

**Comment by BinaryFissionGames** [comment, marked as answer]
Hey @owensulei3841 - You can find the documentation for the key value parser operator here: 
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/pkg/stanza/docs/operators/key_value_parser.md#key_value_parser-operator

You should be able to set the `pair_delimiter` to tab (`\t`) to use a tab instead of whitespace.

**Reply by owensulei3841** [reply]
It works! Thanks a lot!

**Comment by owensulei3841** [comment]
Thanks a lot @BinaryFissionGames
